benefits/core/migrations/local_fixtures.json

```diff
@@ -237,6 +252,27 @@
       "form_class": "benefits.eligibility.forms.SBMTDMobilityPass"
     }
   },
+  {
+    "model": "core.eligibilityverifier",
+    "pk": 7,
+    "fields": {
+      "name": "CalFresh oauth claims via Login.gov",
```

Comment:
Nit, but can we stick with the pattern of prefixing the agency slug e.g. `(MST)`

Reply:
Just want to get this one in @angela-tran and then this PR looks G2G!

Reply:
Forgot about this, thank you!

[5a58a8b](https://github.com/cal-itp/benefits/pull/1958/commits/5a58a8be12f3a3ec530d7d72d501bb686b927b04)